spec/features/work_packages/new_work_package_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'features/work_packages/details/inplace_editor/work_package_field'
+require 'features/work_packages/work_packages_page'
+
+describe 'new work package', js: true, retry: 3, retry_wait: 5 do
+  let(:type_task) { FactoryGirl.create(:type_task) }
+  let(:types) { [type_task] }
+  let(:status) { FactoryGirl.build(:status, is_default: true) }
+  let(:priority) { FactoryGirl.build(:priority, is_default: true) }
+  let(:project) {
+    FactoryGirl.create(:project, types: types)
+  }
+
+  let(:user) { FactoryGirl.create :admin }
+  let(:work_packages_page) { WorkPackagesPage.new(project) }
+
+  let(:subject) { 'My subject' }
+  let(:description) { 'A description of the newly-created work package.' }
+
+  let(:subject_field) { WorkPackageField.new(page, :subject) }
+  let(:description_field) { WorkPackageField.new(page, :description) }
+
+  before do
+    status.save!
+    priority.save!
+
+    login_as(user)
+    allow(user.pref).to receive(:warn_on_leaving_unsaved).and_return('0')
+
+    work_packages_page.visit_index
+    work_packages_page.click_toolbar_button 'Work packages'
+
+    within '#tasksDropdown' do
+      click_link 'Task'
+    end
+  end
+
+  it 'shows the create form with the selected type' do
```

Comment:
I'd put this expectation into the other spec. as for integration specs one should cleverly extend existing features rather then creating new ones.